expression/helper.go

```diff
@@ -70,6 +70,12 @@ func getTimeValue(ctx context.Context, v interface{}, tp byte, fsp int) (d types
 		upperX := strings.ToUpper(x)
 		if upperX == CurrentTimestamp {
 			value.Time = types.FromGoTime(defaultTime)
+			if tp == mysql.TypeTimestamp {
+				err := value.ConvertTimeZone(time.Local, ctx.GetSessionVars().GetTimeZone())
```

Comment:
time.Now() get a timestamp, if we take out it's as yy-mm-dd hh:mm:ss value by `types.FromGoTime()`, the result is based on time.Local.
so the `ConvertTimeZone` operation is from time.Local to UTC